### PR TITLE
remove coc parts that dont make sense on a online event

### PR DESCRIPTION
**Pull request by tlm06**

### CODE REVIEW

File: djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md [hunk 1]
```diff
@@ -102,8 +101,6 @@ In your report please include, when possible:
 
 If you don't have some of this information, or not at this time, please still make the report anyways. You can contact us at any time after your report if you want to add, edit or take back any information you shared.
 
-If you feel unsafe reporting in person, you may choose someone to represent you. In this case, we'd need their contact information, but we'd ask this person to make clear that they are not reporting in their own name.
```
Comment (ewjoachim) [inline djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md]: Why is this not relevant anymore ?

Comment (tlm06) [reply djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md]: after some discussion, we decided to keep that option for someone who wants to make a report so we will add that text again to our CoC

File: djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md [hunk 2]
```diff
@@ -53,7 +53,6 @@ This Code of Conduct applies to all conference related spaces. That includes, bu
 * The conference online platform
 * Any conference related social activities
 * [Slack](https://join.slack.com/t/djangoconeurope2020/shared_invite/zt-cyuhq1vr-XNFnxqD63YGTCl_YdwUJzw){:target="_blank"} channels, tweets with the conference hashtag, and other online media
-* The Code of Conduct does not exclusively apply to events on the conference agenda. For example, if after a scheduled social event, you go to a bar with a group of fellow participants and someone harasses you there, we would still treat that as a CoC violation.
```
Comment (ewjoachim) [inline djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md:56]: What about:

* The Code of Conduct does not exclusively apply to events on the conference agenda. For example, if you join a group of attendees for unofficial virtual drinks after the talks and someone harasses you there, we would still treat that as a CoC violation.

?

Comment (tlm06) [reply djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md:56]: yeah that makes sense... i'll add it 

thank you

**Pull request merged**